.changeset/dry-ligers-end.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+-**chore**: Update Playground title and Github link. By @cpcramer #1081
```

Comment:
No need of this in the Changelog for Playground stuff.

Reply:
Changelog are only for informations that are useful for our developers. This is not a record of the work we did.

Reply:
See Changelog as documentation. Something people need to read so they know what's going on.

cc @cpcramer @0xAlec @abcrane123

Reply:
Got it! 👍